eth-contracts/contracts/InitializableV2.sol

```diff
@@ -16,15 +16,14 @@ contract InitializableV2 is Initializable {
     string private constant ERROR_ALREADY_INITIALIZED = "ERROR_ALREADY_INITIALIZED";
 
     function initialize() public initializer {
-        require(!initialized, ERROR_ALREADY_INITIALIZED);
```

Comment:
removed this because this can never be hit, it will always be set in the initializer modifier